prov/rxd/src/rxd.h

```diff
@@ -219,6 +219,11 @@ struct rxd_ep {
 	struct rxd_peer peers[];
 };
 
+static inline struct rxd_peer *rxd_ep_getpeer(struct rxd_ep *ep, 
```

Comment:
nit: To limit how much this change increases line lengths how about just rxd_peer or rxd_getpeer?  Otherwise, I'm going to ask you to check the line lengths for several of the changes below and fix them up.  :)